packages/okta-vue/test/e2e/harness/package.json

```diff
@@ -6,14 +6,16 @@
   "private": true,
   "scripts": {
     "dev": "webpack-dev-server --inline --progress --config build/webpack.dev.conf.js",
-    "start": "npm run dev",
+    "start": "npm run env && npm run dev",
+    "env": "node ./scripts/prebuild.js",
```

Comment:
nit: To avoid the `&&` operator, we should make this a `prestart` and `pretest` script.

Reply:
IMO I actually find it harder to fish through the scripts map and find all the pre/post scripts.  I know we do this everywhere and that it's a node thing so I'm not saying we should change course, but just want to put my .02 in

Reply:
Yeah - I think I was a bit confused by what `env` was doing by the name. Maybe we can rename it to: `setup-env-vars`?